apps/web/pages/_app.tsx

```diff
@@ -7,7 +9,17 @@ import "../styles/globals.css";
 function MyApp(props: AppProps) {
   const { Component, pageProps } = props;
   if (Component.PageWrapper !== undefined) return Component.PageWrapper(props);
```

Comment:
Do we not need to also do this in the case this line? i.e. When a page uses PageWrapper?

Reply:
not really sure to be honest

Reply:
I think we do 👍